sql/moz-fx-cjms-prod-f3c7/cjms_bigquery/subscriptions_v1/query.sql

```diff
@@ -0,0 +1,68 @@
+WITH aic_flows AS (
+  -- last fxa_uid for each flow_id
+  SELECT
+    flow_id,
+    MIN(flow_started) AS flow_started,
+    ARRAY_AGG(fxa_uid IGNORE NULLS ORDER BY fxa_uid_timestamp DESC LIMIT 1)[
+      SAFE_OFFSET(0)
+    ] AS fxa_uid,
```

Comment:
I discussed with product, and we don't care about edge cases with earlier `fxa_uid`s during the MVP